.github/workflows/test.yml

```diff
@@ -50,21 +49,39 @@ jobs:
         run: cargo clippy --locked -- -D warnings
 
       - name: Test generating shell completion files
-        if: matrix.os == 'ubuntu-latest'
+        if: matrix.os == 'ubuntu-20.04'
         run: cargo xtask gencomp
```

Comment:
I guess we're keeping this here because it depends on CLI?

Though tbh the step confuses me a bit since it just generates the completions, which doesn't really seem helpful to me.

Reply:
Just to be clear what I mean here: For docs generation we have an actual test that ensures the docs are up-to-date. That seems much more sensible.

Reply:
Good point. I've written an actual test to replace this.